libclambcc/ClamBCChangeMallocArgSize/ClamBCChangeMallocArgSize.cpp

```diff
@@ -1,162 +1,182 @@
 
 #include <llvm/Pass.h>
-#include "llvm/IR/Function.h"
-#include "llvm/IR/Module.h"
-#include "llvm/IR/Instructions.h"
-#include "llvm/Support/raw_ostream.h"
+#include <llvm/IR/Function.h>
+#include <llvm/IR/Module.h>
+#include <llvm/IR/Instructions.h>
+#include <llvm/Support/raw_ostream.h>
 
-#include "llvm/IR/LegacyPassManager.h"
-#include "llvm/Transforms/IPO/PassManagerBuilder.h"
+#include <llvm/IR/LegacyPassManager.h>
+#include <llvm/Transforms/IPO/PassManagerBuilder.h>
+
+#include <llvm/Passes/PassBuilder.h>
+#include <llvm/Passes/PassPlugin.h>
 
 #include "Common/clambc.h"
 
 using namespace llvm;
 
-namespace
-{
-class ChangeMallocArgSize : public ModulePass
-{
-  protected:
-    std::vector<PHINode*> changeValues;
+namespace ChangeMallocArgSize {
+    class ChangeMallocArgSize : public PassInfoMixin<ChangeMallocArgSize >
+    {
+        protected:
+            std::vector<PHINode*> changeValues;
```

Comment:
Your change in formatting in this file seems nice, but is contrary to the `.clang-format` spec, and makes it a little harder to review the changes to the code since I'm mostly looking at a diff of formatting changes.

Please run the `.clam-format` script.

Reply:
done.